.gitattributes

```diff
@@ -5,3 +5,4 @@ tests/data/plugins/browsers/firefox/places.sqlite filter=lfs diff=lfs merge=lfs
 tests/data/plugins/browsers/chrome/History.sqlite filter=lfs diff=lfs merge=lfs -text
 tests/data/plugins/browsers/edge/History.sqlite filter=lfs diff=lfs merge=lfs -text
 tests/data/plugins/browsers/chromium/History.sqlite filter=lfs diff=lfs merge=lfs -text
+tests/data/plugins/os/unix/log/journal filter=lfs diff=lfs merge=lfs -text
```

Comment:
I see that we are not really giving a good example here but would it be possible to make a journal file for testing purposes? Since you seem to only use 1 record 8 MB seems a bit much. Also, are you sure no personal data is in the binary blob? I assume it comes from your working station? If you can make a test-only systemd journal it also does not have to go to LFS (probably).